### PR TITLE
Kat10140/calcite shell floating panel rename

### PR DESCRIPTION
**Pull request by kat10140**
**Related Issue:** #183

### CODE REVIEW

**Comment by kat10140** [comment]
@driskull looks like there are a few things that are in fact changed, but are not showing up in the change log? They are showing up on my branch on here. 🤷‍♀

File: src/components/calcite-shell-panel/readme.md [hunk 1]
```diff
@@ -1,28 +1,26 @@
 # calcite-shell-panel
 
-The `calcite-shell-panel` is a child component of `calcite-shell` used to house and display other components like `calcite-block`, `calcite-flow` and `floating-panel`.
+The `calcite-shell-panel` is a child component of `calcite-shell` used to house and display other components like `calcite-block`, `calcite-flow` and `calcite-shellfloating-panel`.
```
Comment (driskull) [inline src/components/calcite-shell-panel/readme.md]: needs a dash here. `calcite-shell-floating-panel`

Comment (kat10140) [reply src/components/calcite-shell-panel/readme.md]: 👍 done

**Comment by kat10140** [comment]
@driskull should I merge this? Is it normal to merge your own PRs?

**Comment by driskull** [comment]
> looks like there are a few things that are in fact changed, but are not showing up in the change log? They are showing up on my branch on github. 🤷‍♀

Where are these showing up @kat10140 ? what files

**Comment by kat10140** [comment]
> > looks like there are a few things that are in fact changed, but are not showing up in the change log? They are showing up on my branch on github. 🤷‍♀
> 
> Where are these showing up @kat10140 ? what files

in the calcite-shell demo

**Comment by driskull** [comment]
@kat10140 those are ok. they are in the `docs` folder which gets built with `npm run docs`. We can do that after merge. 

good to merge

**Comment by kat10140** [comment]
gotcha. thanks @driskull !

**Pull request merged**